eZ/Publish/Core/Persistence/Legacy/Content/Handler.php

```diff
@@ -239,6 +250,46 @@ public function publish($contentId, $versionNo, MetadataUpdateStruct $metaDataUp
         return $this->load($contentId, $versionNo);
     }
 
+    protected function copyTranslations(int $contendId, VersionInfo $versionInfo): void
+    {
+        $draft = $this->load($contendId, $versionInfo->versionNo);
+        $publishedContent = $this->load($contendId);
+        $publishedVersionInfo = $publishedContent->versionInfo;
+
+        if ($versionInfo->versionNo >= $publishedVersionInfo->versionNo) {
+            return;
```

Comment:
Ok, now I get it.
we need some explicit inline doc above `if`
e.g. 
```php
// copying occurs only if Version being published is older than the currently published one
```
Also please move 1st line (`$draft = $this->load($contendId, $versionInfo->versionNo)`) after this check to avoid unnecessary load if condition is not met.